<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userGUIClass</class>
 <widget class="QMainWindow" name="userGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>userGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="filterByPresenterLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="filteredByPresenterList"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="addToWatchListButton">
        <property name="text">
         <string>Add to Watch List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteFromWatchListButton">
        <property name="text">
         <string>Delete From Watch List</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="seeWatchListButton">
        <property name="text">
         <string>See the Watch List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openOnYoutubeButton">
        <property name="text">
         <string>Watch video on YouTube</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
